### ISSUE TITLE
Live Stream and User Groups Issue

### ISSUE THREAD

**Issue by guymass** [WWBN/AVideo#7070]
Lately we noticed that one of our Admin users when he makes a Live Stream in the end when that stream is encoded it will also get added to User Groups, this will cause the stream to be viewed only by users who are in those groups. We are looking where in the setting we can remove this or edit this feature as we want the live stream to be public also to users who are not logged in.

Here is a screen shot of an encoded video where you can see (sorry its in hebrew) that that video has groups added to it:

![Screen Shot 5782-13-05 at 11 49 12](https://user-images.githubusercontent.com/58882903/187874010-4b5a4900-81d7-49ff-89f1-1a7daa60ba98.jpg)

It is the fifth field from top. How can we remove those groups from the encoded stream so it doesn't appear anymore?

**Comment by guymass** [comment]
Also I am in English mode on the site and in those video details I can still see five fields which are still presented as Hebrew while they all should be English.

![Screen Shot 5782-13-05 at 15 27 56](https://user-images.githubusercontent.com/58882903/187913898-df672454-62e4-4257-af86-baccefbbb9f4.jpg)

**Comment by DanielnetoDotCom** [comment]
please check the PredefinedCategory plugin

![chrome-capture-2022-8-1](https://user-images.githubusercontent.com/18009293/187930409-3009416f-18b8-41af-a49d-47dbf4172453.gif)

**Comment by Shaichu** [comment]
please guys its been 3 weeks now that my audience cannot enter live broadcasts, its only on my admin account which is realy weird. please sort it out asap. i cannot go live whiche is the whole puprose of avideo!
thanks

**Comment by Shaichu** [comment]
hi again, we need a solution. something does not work properly

**Comment by DanielnetoDotCom** [comment]
To be able to resolve it I need to reproduce this behavior. 

can you reproduce it on the demo site?

**Comment by Shaichu** [comment]
i hope Guy can do it, i am not on the tech side. Guy?

**Comment by guymass** [comment]
I will try I just don't know how it came to be. I will post when I am able to reproduce

**Comment by guymass** [comment]
OK here is something I found. On our main site, in the Go Live page under the stream settings it look like this:
Only Stream Meta Data and Poster tabs appear.

![Screen Shot 5782-13-16 at 9 36 49](https://user-images.githubusercontent.com/58882903/189588769-4f419f27-0463-45ff-9e42-f50af5087290.jpg)


On my second test site I have an additional tab for selecting those groups:
![Screen Shot 5782-13-16 at 9 37 06](https://user-images.githubusercontent.com/58882903/189588990-f0323aec-ddb4-4930-96b0-f33b8959286b.jpg)


So basically this third tab for User Groups is missing on my main site. Kinda strange but that is the case.

How do I fix that?

**Comment by guymass** [comment]
OK under Live plugin there is an option to hide those user groups and it was checked, I unchecked it and now those groups appear as the third tab.

![Screen Shot 5782-13-16 at 9 45 14](https://user-images.githubusercontent.com/58882903/189589908-2d0c27cc-94b6-4592-9e29-ffc24c72016d.jpg)

I will test to see if now I can disable those groups for the channel's live stream and see if this now works.

**Comment by guymass** [comment]
Yes, that solved the problem! it appears that those groups were "Marked" but the user wasn't able to access that third tab to edit them and thus each stream was automatically added to those selected groups. Once the tab was back we were able to unselect those groups and now the stream settings are working as they should.

![Screen Shot 5782-13-16 at 9 50 26](https://user-images.githubusercontent.com/58882903/189591021-6fcc2cf3-1716-4adc-b9d8-6f49ae9e5f6d.jpg)
